packages/rln/src/contract/rln_contract.spec.ts

```diff
@@ -1,82 +1,277 @@
-import { expect } from "chai";
+import { hexToBytes } from "@waku/utils/bytes";
+import { expect, use } from "chai";
+import chaiAsPromised from "chai-as-promised";
 import * as ethers from "ethers";
+import sinon, { SinonSandbox } from "sinon";
 
 import { createRLN } from "../create.js";
+import type { IdentityCredential } from "../identity.js";
 
 import { SEPOLIA_CONTRACT } from "./constants.js";
 import { RLNContract } from "./rln_contract.js";
 
-describe("RLN Contract abstraction", () => {
-  it("should be able to fetch members from events and store to rln instance", async () => {
+use(chaiAsPromised);
+
+// Use the minimum allowed rate limit from RATE_LIMIT_TIERS
+const DEFAULT_RATE_LIMIT = 20;
+
+describe("RLN Contract abstraction - RLN v2", () => {
```

Comment:
no need to specify `rln v2`, there is no other `rln` that will be supported in this code base